<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplacementWindowClass</class>
 <widget class="QWidget" name="DisplacementWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisplacementWindow</string>
  </property>
  <widget class="QPushButton" name="pushButtonSure">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>适用</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDX">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DX</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDXY">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DXY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDZ">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDYZ">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DYZ</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDY">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DY</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDXZ">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DXZ</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDXYZ">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>DXYZ</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
